extensions/jt400/deployment/src/main/java/org/apache/camel/quarkus/component/jt400/deployment/Jt400Processor.java

```diff
@@ -86,6 +89,16 @@ void reflectiveClasses(BuildProducer<ReflectiveClassBuildItem> reflectiveClasses
 
     }
 
+    @BuildStep
+    void resourceBundles(BuildProducer<NativeImageResourceBundleBuildItem> imageResourceBundles,
+            CombinedIndexBuildItem combinedIndex) {
+        IndexView index = combinedIndex.getIndex();
+
+        index.getAllKnownSubclasses(LIST_RESOURCE_BUNDLE_NAME).stream()
```

Comment:
Probably best to filter on a package prefix of `com.ibm.as400`.

Otherwise we register everything on the classpath extending `ListResourceBundle`. Which is probably not what we want.

Reply:
fixed